### DISCUSSION TITLE
this block is missing its respective building, try restarting or loading a backup issue

### DISCUSSION THREAD

**Discussion by Themexicantaco** [ldtteam/minecolonies#10022] [Q&A]
Hello! Im kinda new to mine colonies. I built my warehouse and then built the couriers hut but when I place it down, it says "
this block is missing its respective building, try restarting or laoding a backup issue". Im not sure what to do.

**Comment by MotionlessTrain** [comment]
Please use [discord](https://discord.minecolonies.com) for mod support!

Make sure to build buildings with the build tool. Without using a build tool, they won't work
You can right click the courier's hut in the air to see where you can find it in the build tool, in each style. Some styles have it integrated in the warehouse, in such case you may need to find the courier's hut block, and order it to be built there
